config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -360,27 +360,27 @@ periodics:
 
 # manual-release-bump-required
 - name: ci-kubernetes-node-kubelet-stable3
-  interval: 24h
+  interval: 6h
   labels:
     preset-service-account: "true"
     preset-k8s-ssh: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.10
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.11
       args:
-      - --repo=k8s.io/kubernetes=release-1.10
+      - --repo=k8s.io/kubernetes=release-1.11
       - --timeout=90
       - --root=/go/src
       - --scenario=kubernetes_e2e
       - --
       - --deployment=node
       - --gcp-project=k8s-jkns-ci-node-e2e
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-1-10.yaml
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-1-11.yaml
       - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
       - --node-tests=true
       - --provider=gce
-      - --test_args=--nodes=8 --skip="\[Flaky\]|\[Serial\]"
+      - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Serial\]"
```

Comment:
Not sure I follow why the change in focus here

Reply:
it flows down from the previous stable2 job